### PR TITLE
Membership QA fixes

### PR DESCRIPTION
**Pull request by FrederikBolding**
- Fixed styling for `ContentPanel` based on design meeting
- Automatic fetching of membership state after purchase
- Improved membership fetching and handling overall
- Fixed small styling and copy issues for membership

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch5979]

**Comment by FrederikBolding** [comment]
Will be bumping `ui` lib in this PR as well, so please hold off until that has been done.

**Comment by FrederikBolding** [comment]
Should be good to go now 😄

File: common/v2/services/Store/StoreProvider.tsx [hunk 1]
```diff
@@ -209,30 +211,25 @@ export const StoreProvider: React.FC = ({ children }) => {
...
-
-    return () => {
-      isMounted = false;
-    };
-  }, [currentAccounts]);
+  };
 
   useEffect(() => {
     setPendingTransactions(getPendingTransactionsFromAccounts(currentAccounts));
+    setIsMembershipScanRequested(true);
```
[the remark this answers is not in the document]
Comment (FrederikBolding) [reply common/v2/services/Store/StoreProvider.tsx:232]: We want to check for membership status if a new account is added. I can make it its own `useEffect` if that's clearer. It just seemed weird to do for one line 😄

[the remark this answers is not in the document]
Comment (blurpesec) [reply common/v2/services/Store/StoreProvider.tsx:232]: Maybe change it to scan the 1 account, instead of triggering a scan of all accounts? :-0

**Pull request merged**